docs/reference/modules/scripting/using.asciidoc

```diff
@@ -240,3 +239,12 @@ NOTE: The size of stored scripts is limited to 65,535 bytes. This can be
 changed by setting `script.max_size_in_bytes` setting to increase that soft
 limit, but if scripts are really large then alternatives like
 <<modules-scripting-native,native>> scripts should be considered instead.
+
+[[modules-scripting-using-caching-note]]
```

Comment:
Rather than providing this info in a note, I would suggest having a short section called something like "Script Caching." Putting the content in a note sort of buries it.